source/include/ble/cs_Characteristic.h

```diff
@@ -82,7 +82,7 @@ class CharacteristicBase {
 public:
```

Comment:
In general I think the implementation details in this header should move to a .cpp file, and maybe we should schedule a general facelift for this file. :-)

Reply:
Also for later.